### PR TITLE
Update modals, prompt and dependencies

### PR DESCRIPTION
**Pull request by sdserranog**
Sync the Main branches between the repos.

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -7,8 +7,10 @@ import './globals.css';
 
 export const metadata: Metadata = {
   metadataBase: new URL('https://arcade-ai.com'),
-  title: 'Arcade AI Chatbot',
-  description: 'Explore and enjoy testing your tools with our platform.',
+  title: 'Arcade AI Chat',
+  description:
+    'hat with Archer, an AI assistant built by Arcade AI with access to \
```
Comment (nbarbettini) [inline app/layout.tsx]: is the first word supposed to be `Chat`? or is the `C` rendered elsewhere?

File: ai/prompts.ts [hunk 2]
```diff
@@ -1,3 +1,11 @@
-export const regularPrompt = 'You are a friendly assistant!';
+export const regularPrompt =
+  'You will play the role of Archer, a highly knowledgeable AI assistant \
+with a playful and witty personality. As of ${new Date().toLocaleDateString()}, engage \
+Engage in conversation with the user, providing informative and helpful responses while \
```
Comment (nbarbettini) [inline ai/prompts.ts]: nit: `engage` is duplicated

File: components/custom/multimodal-input.tsx [hunk 3]
```diff
@@ -21,9 +21,20 @@ import { Textarea } from '../ui/textarea';
 
 const suggestedActions = [
   {
-    title: 'Start the',
-    label: 'ArcadeAI/arcade-ai repo',
-    action: 'Start the ArcadeAI/arcade-ai repo',
+    title: 'What actions can you take for me?',
+    label: 'supported by Arcade AI',
```
Comment (nbarbettini) [inline components/custom/multimodal-input.tsx]: nit: we say `powered by Arcade AI` in other places in our docs

File: app/layout.tsx (hunk 1 above)
Comment (sdserranog) [reply app/layout.tsx]: Thank you, Nate, for catching these typos

**Review by nbarbettini — APPROVED** [review]
LGTM!

**Pull request merged**